scripts/qa_crowbarsetup.sh

```diff
@@ -3546,8 +3546,10 @@ function glance_image_exists()
 
 function glance_image_get_id()
 {
-    local image_id=$(openstack image list | grep "[[:space:]]$1[[:space:]]" | awk '{ print $2 }')
-    echo $image_id
+    id=""
```

Comment:
not needed here, because it is running in a subshell... but we could still add it for consistency